packages/flutter_tools/test/commands.shard/hermetic/build_web_test.dart

```diff
@@ -116,6 +116,46 @@ void main() {
     }),
   });
 
+  testUsingContext('Builds a web bundle in specified directory - end to end',
+      () async {
+    final BuildCommand buildCommand = BuildCommand();
+    final CommandRunner<void> runner = createTestCommandRunner(buildCommand);
+
+    setupFileSystemForEndToEndTest(fileSystem);
+
+    const String newBuildDir = 'new_dir';
+    final Directory buildDir = fileSystem.directory(fileSystem.path.join(newBuildDir));
+
+    await runner.run(<String>[
+      'build',
+      'web',
+      '--no-pub',
+      '--output-dir=$newBuildDir'
+    ]);
+
+    expect(buildDir.existsSync(), true);
```

Comment:
can you also check some of the contents of the build dir?

Reply:
I'm not sure the test setup actually produces anything in the build directory. When I use the debugger to inspect the contents of that directory, it looks like it is empty. I followed the same setup used for this test shown here

https://github.com/eliasyishak/flutter/blob/specify-build-output-directory/packages/flutter_tools/test/commands.shard/hermetic/build_web_test.dart#L87:L117

Should I continue digging into the test build system or do you think this is enough?

Reply:
so are we just testing that the buildDir gets created? do you know why we don't put anything in it?

Reply:
yes, just finished chatting with @jonahwilliams and he suggests that we test that the build directory is correctly getting created and configured and allow the build system and its tests to correctly identify any problems that come during the build

Reply:
ok, can you update the test name to reflect what we're actually testing here (which seems to me to be that the buildDir is created at the correct location)?